cms/bundles/tests/test_forms_with_dataset_api.py

```diff
@@ -621,6 +626,8 @@ def test_dataset_validation_multiple_datasets_mixed_statuses(self):
         """Test validation with multiple datasets having different statuses."""
         dataset1 = DatasetFactory(id=123, title="Approved Dataset")
         dataset2 = DatasetFactory(id=124, title="Draft Dataset")
+        bundled_dataset1 = BundleDatasetFactory(parent=self.bundle, dataset=dataset1)
+        bundled_dataset2 = BundleDatasetFactory(parent=self.bundle, dataset=dataset2)
```

Comment:
Nit: Shouldn't it be `bundled_dataset_1` and `bundled_dataset_2`? Unless in this test file, we are going with no underscores. I think we are inconsistent with this. It would improve legibility and feel more Pythonic. I know `pep8`doesn't strictly speak about digits in variable naming.

Reply:
Agreed, it was to be consistent within the existing method. Happy to change.